administrator/language/de-DE/com_guidedtours.sys.ini

```
@@ -0,0 +1,376 @@
+
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURSTEPS_STEP_TYPE_TITLE="Den Typ auswählen"
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURSTEPS_STEP_TYPE_DESCRIPTION="Dies ist die Art von Schritt, die erstellt werden soll. 'Weiter', um die Tour ohne Interaktion fortzusetzen, 'Weiterleiten', um zu einer anderen Seite zu gelangen, 'Interaktion', um Benutzereingaben zu verlangen."
+
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURSTEPS_STEP_SAVECLOSE_TITLE="'Speichern &amp; Schließen' wählen"
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURSTEPS_STEP_SAVECLOSE_DESCRIPTION="Schritt speichern und schließen."
+
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURSTEPS_STEP_CONGRATULATIONS_TITLE="Herzlichen Glückwunsch!"
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURSTEPS_STEP_CONGRATULATIONS_DESCRIPTION="Ein neuer Schritt für eine geführte Tour wurde erstellt."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_TITLE="Wie erstellt man Beiträge?"
+COM_GUIDEDTOURS_TOUR_ARTICLES_DESCRIPTION="Diese Tour zeigt, wie man einen Beitrag erstellt."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_NEW_TITLE="'Neu' wählen"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_NEW_DESCRIPTION="Schaltfläche 'Neu' wählen, um einen Beitrag zu erstellen."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_TITLE_TITLE="Titel für den Beitrag vergeben"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_TITLE_DESCRIPTION="Den gewünschten Titel für den Beitrag eingeben."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_ALIAS_TITLE="Einen Alias eingeben"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_ALIAS_DESCRIPTION="Den internen Namen für diesen Beitrag eingeben. Wird dieses Feld leer gelassen, dann wird ein Standardwert in Kleinbuchstaben mit Bindestrichen anstelle von Leerzeichen als Teil der URL verwendet."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_CONTENT_TITLE="Beitragsinhalt hinzufügen"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_CONTENT_DESCRIPTION="Den Inhalt des Beitrags in den Editor eingeben."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_STATUS_TITLE="Status wählen"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_STATUS_DESCRIPTION="Den Status des Beitrags auswählen."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_CATEGORY_TITLE="Kategorie wählen"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_CATEGORY_DESCRIPTION="Die Kategorie für diesen Beitrag auswählen. Es kann auch direkt eine neue Kategorie angelegt werden, indem man den Namen in das Feld eingibt und mit der Eingabetaste bestätigt."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_FEATURED_TITLE="Status 'Haupteintrag' wechseln"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_FEATURED_DESCRIPTION="'Ja' wählen, um den Beitrag als Haupteintrag zu kennzeichnen. Dies kann z.&#8239;B. für Sortierungen verwendet werden."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_ACCESS_TITLE="Zugriffsebene wählen"
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_ACCESS_DESCRIPTION="Legt die Zugriffsebene für den Beitrag fest."
+
+COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_TAGS_TITLE="Schlagwörter wählen"
```

```suggestion
COM_GUIDEDTOURS_TOUR_ARTICLES_STEP_TAGS_TITLE="Schlagwörter hinzufügen"
```
wird bereits 3x verwendet